app/controllers/insured/plan_shoppings_controller.rb

```diff
@@ -205,6 +205,11 @@ def plans
 
   private
 
+  # no dental as of now
+  def sort_member_groups(products)
```

Comment:
Updated sort method so that the _products array_ has **current selected product** as the first element.